### PR TITLE
Add param description to `fileMove()`

### PR DESCRIPTION
**Pull request by shaedrich**

### CODE REVIEW

File: data/en/filemove.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
 	"links": [], 
     	"examples": [
 		{
-			"title": "Script Syntax",
+			"title": "Move file from here to there",
 			"description": "",
-			"code": "<cfscript>\r\nfileMove(#sourcefile#, #destinationfile#);\r\n</cfscript>",
+			"code": "fileMove(#sourcefile#, #destinationfile#);",
```
Comment (pfreitag) [inline data/en/filemove.json]: Thanks for removing cfscript - The pound signs can be removed here too, so just `fileMove(sourcePath, destinationPath)`

Comment (shaedrich) [reply data/en/filemove.json]: Sure. Looks like I have been a bit inattentive^^

**Comment by pfreitag** [comment]
Thanks!

**Comment by shaedrich** [comment]
You're welcome!

**Pull request merged**